src/Npgsql/Replication/PgOutput/Messages/LogicalDecodingMessage.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using NpgsqlTypes;
+
+namespace Npgsql.Replication.PgOutput.Messages
+{
+    /// <summary>
+    /// Logical Replication Protocol logical decoding message
+    /// </summary>
+    public sealed class LogicalDecodingMessage : PgOutputReplicationMessage
```

Comment:
Any idea what this new message actually means? :)

Reply:
Yeah, it reads whatever is sent via [pg_logical_emit_message](https://www.postgresql.org/docs/current/functions-admin.html#id-1.5.8.33.8.5.2.2.22.1.1.1)

Reply:
What's a real-world use for this? Just trying to understand the new feature :)

Reply:
As I understand it, you can send arbitrary messages or data through the replication stream.
I think the original idea might have been to configure logical decoding plugins via SQL but in the Npgsql case you can pass anything you enter via SQL through the steam and read it on the other end.
I hope it will become more clear once we have tests to showcase the feature.